core/win32/callback.c

```diff
@@ -5854,6 +5900,10 @@ initialize_exception_record(EXCEPTION_RECORD* rec, app_pc exception_address,
         rec->ExceptionInformation[0] = EXCEPTION_EXECUTE_FAULT /* execution tried */;
         rec->ExceptionInformation[1] = (ptr_uint_t)exception_address;
         break;
+    case SINGLE_STEP_EXCEPTION:
+        rec->ExceptionCode = EXCEPTION_SINGLE_STEP;
+        rec->NumberParameters = 0;
+        break;
```

Comment:
The new issue can be opened and a FIXME or XXX i# ref put here.  But I agree that *implementing* it for UNIX should be separate.